assets/Technology-Tag-Icons/icons.tsx

```diff
@@ -0,0 +1,143 @@
+      fill={props.fill}
+    />
+  </svg>
+);
+
+export const LandBasedWindIcon = (props: {
+  fill: string | undefined;
+  stroke: string | undefined;
```

Comment:
if **stroke** is not passed into this icon, we don't need to have a stroke prop! You can remove this.